paparazzi/paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -622,6 +622,15 @@ class Paparazzi @JvmOverloads constructor(
     private val isVerifying: Boolean =
       System.getProperty("paparazzi.test.verify")?.toBoolean() == true
 
+    private val hasComposeRuntime: Boolean =
```

Comment:
How often does this change? Should it be cached either `by lazy` or in `companion object`?

Reply:
Will be calculated for each new Paparazzi rule creation for each test method run.  Fine for now, unless it's quick to profile and improve via companion object.

Reply:
Oh, this is already in `companion object`.  Never mind!

Reply:
Oops 🤐, need to expand context on GH 😞